### DISCUSSION TITLE
Question: is it possible to use this with `dprint` and how

### DISCUSSION THREAD

**Discussion by stevenliebregt** [tree-sitter/tree-sitter#3113] [Q&A] [answered]
Hi, I'm attempting to make a WASM plugin for `dprint` that uses `tree-sitter` and the `tree-sitter-sql` plugin, but I'm having a ton of trouble trying to compile it to WASM. I found https://crates.io/crates/tree-sitter-wasm-build-tool, that seemed to start working, but then the clang compilation of the plugin went out of memory (21 GB +). Now I'm wondering, is it even possible to compile this to WASM for usage with `dprint`?

**Comment by ObserverOfTime** [comment, marked as answer]
It should be now: #4820
